src/components/HolidayForm.js

```diff
@@ -46,46 +43,23 @@ const StyledHolidayForm = styled(HolidayForm)`
   color: ${(props) => props.theme.color.majorBrown};
 
   .center-form {
-    margin: 0 auto;
     display: flex;
-    flex-flow: column;
+    flex-direction: column;
     justify-content: space-between;
     align-items: center;
-
-    ${(props) => props.theme.media.tablet} {
-      flex-flow: row;
-      width: 70%;
-    }
-
-    ${(props) => props.theme.media.desktop} {
-      width: 60%;
+    padding: 1rem 0;
+    margin: 0 auto;
+    > * {
```

Comment:
Can you help me understand how the combinator works here? I vaguely know what it does but never used in my own code. 🙏

Reply:
This selects all child elements recursively, you can find more details [here](https://www.geeksforgeeks.org/how-to-select-all-child-elements-recursively-using-css/)  :)